extension/src/nodes/node.cpp

```diff
@@ -0,0 +1,20 @@
+#include "node.h"
```

Comment:
I'd recommend avoid using "node", since it is already taken by Godot. 
Also, if you'd like to follow [C++ naming convention](https://manual.gromacs.org/5.1.1/dev-manual/naming.html#files), then I'd recommend using the same file name as the class name - `BeehaveNode.ccp` and `BeehaveNode.h` in this case.